Reduce the length of the frameworks page by half (#230)
open-contracting-extensions/public-private-partnerships: docs/framework.md

</div>

> A reference to the sponsoring agency or department should be provided using an [OrganizationReference](https://standard.open-contracting.org/1.1/en/schema/reference/#organizationreference) object in the [`publicAuthority`](../reference/schema/#release-schema.json,,publicAuthority) field, referencing the relevant entry in the `parties` section.

Reviewer: It's not clear that `publicAuthority` is a link, since it has the same styling as other non-linked field names.

Reviewer: It would be better for the link to the parent field to be before the link to the object reference, since the former includes the content from the latter, so the reader need only click one link instead of two to get the information that was originally included in the schema tables.

Author: Once a user learns what an OrganizationReference is, they don't need to follow that link anymore. This construction is used everywhere for that reason: provide the more fundamental information first.

If we change it here we should change it everywhere, but I think the sentence flows better as-is, and it introduces more fundamental concepts first. Users only need to click twice the first time they see an OrgRef link.

</div>

The value, or range of values, anticipated during the planning stage. 

> **Schema**: Information can be provided using a [Value](https://standard.open-contracting.org/1.1/en/schema/reference/#value) object in the [`planning/project/totalValue`](../reference/schema/#release-schema.json,,planning/project/totalValue) field.

Reviewer: It's not clear that `planning/project/totalValue` is a link, since it has the same styling as other non-linked field names. Applies to all other linked field names.

Author: Fixed 0f8979b

Key events relating to the implementation of the project.

> This information can be provided as [Milestone](https://standard.open-contracting.org/1.1/en/schema/reference/#milestone) objects in the [`contracts/milestones`](../reference/schema/#release-schema.json,,contracts/0/milestones) array, each with its `type`, `code` and `status` fields set to codes and its `documents` array set to links to any additional documentation.

Reviewer: Clarify where the codes come from:

```suggestion
This information can be provided as [Milestone](https://standard.open-contracting.org/1.1/en/schema/reference/#milestone) objects in the [`contracts/milestones`](../reference/schema/#release-schema.json,,contracts/0/milestones) array, each with its `type`, `code` and `status` fields set to codes from the relevant codelists and its `documents` array set to links to any additional documentation.
```

Author: Fixed 0f8979b (and other occurrence)

State the actual annual measured levels of demand or stated levels of demand in the providers report or contract managers report. Use the following format

Structured data about estimated demand should be provided using [Metric](https://extensions.open-contracting.org/en/extensions/metrics/master/schema/#metric) objects in the [`contracts/implementation/metrics`](../reference/schema/#release-schema.json,,contracts/0/implementation/metrics) array.

> A metric with an `id` of 'demand' should be given, with a series of [`observations`](../reference/schema/#release-schema.json,,contracts/0/implementation/metrics/0/observations) that capture the actual demand for a given period. These estimates can be disaggregated by any number of dimensions contained as simple fields within each observation's [`dimensions`](../reference/schema/#release-schema.json,,contracts/0/implementation/metrics/0/observations/0/dimensions) object. 

Reviewer: Clarify the meaning of 'simple fields'. Suggest applying to all other occurrences too.

```suggestion
A metric with an `id` of 'demand' should be given, with a series of [`observations`](../reference/schema/#release-schema.json,,contracts/0/implementation/metrics/0/observations) that capture the actual demand for a given period. These estimates can be disaggregated by any number of dimensions contained as key-value pairs within each observation's [`dimensions`](../reference/schema/#release-schema.json,,contracts/0/implementation/metrics/0/observations/0/dimensions) object. 
```

Author: This was present in the original, but happy to make the change.

Author: Fixed 0f8979b

### X.3. Actual IRR 

Recommended only where there is government equity investment or other form of government support that is substantial

Structured data about actual IRR can be provided using [Metric](https://extensions.open-contracting.org/en/extensions/metrics/master/schema/#metric) objects in the [`contracts/implementation/metrics`](../reference/schema/#release-schema.json,,contracts/0/implementation/metrics) array.

A metric with an `id` of 'IRR' should be given, with a series of [`observations`](../reference/schema/#release-schema.json,,contracts/0/implementation/metrics/0/observations) that capture the actual revenue for a given period. These estimates can be disaggregated by any number of dimensions contained as simple fields within each observation's [`dimensions`](../reference/schema/#release-schema.json,,contracts/0/implementation/metrics/0/observations/0/dimensions) object. 

**Example**: See [section X.1](#x-1-actual-annual-demand) for JSON and flattened examples of the `metrics` building block.

### X.4. Actual KPI performance 

State actual year-wise performance here against each of 10-12 identified key performance indicators

> Structured data about actual performance against KPIs be provided using [Metric](https://extensions.open-contracting.org/en/extensions/metrics/master/schema/#metric) objects in the [`contracts/implementation/metrics`](../reference/schema/#release-schema.json,,contracts/0/implementation/metrics) array.

Reviewer: typo

```suggestion
Structured data about actual performance against KPIs can be provided using [Metric](https://extensions.open-contracting.org/en/extensions/metrics/master/schema/#metric) objects in the [`contracts/implementation/metrics`](../reference/schema/#release-schema.json,,contracts/0/implementation/metrics) array.
```

Author: Fixed 0f8979b
